src/party_menu.c

```diff
@@ -3210,126 +3214,126 @@ static void sub_81B2720(u8 windowId)
     BlitBitmapToPartyWindow(windowId, gUnknown_08615A80, 18, 0, 0, 18, 3);
 }
 
-static void UpdateSelectedPartyBox(struct Struct203CEDC *ptr, u8 bitfield)
+static void UpdateSelectedPartyBox(struct Struct203CEDC *ptr, u8 bitfield) //see GetPartyBoxPalBitfield for bitfield
```

Comment:
Can we pull this bitfield out into its own struct? That way we can name each field.
If that doesn't match, then we should at least use macros for the masks.

Reply:
If the type is `u8`, you can't. Bitfields in C are always word-sized.

Reply:
Welp. We should use macros to name the masks, then.

Reply:
Question on that, btw, which may or may not be useful for this particular instance: Do enums get turned into constants they represent in compiled C, for the sake of compilation matching? Could I do an enum for instance, like below, and it'll still match? Or will that cause something about the enum to get generated into the compilation. I tried looking this up online but no one out there cares enough about the compiled output of their code.
```c
enum {
    IS_SELECTED = 1,
    IS_FAINTED = 2,
    // etc
};
```

Reply:
Yes, they are effectively constants in C, and the above enum is functionally identical to 
```
#define IS_SELECTED 1
#define IS_FAINTED 2
```